Formula/notcurses.rb

```diff
@@ -0,0 +1,35 @@
+class Notcurses < Formula
+  desc "Blingful character graphics/TUI library"
+  homepage "https://nick-black.com/dankwiki/index.php/Notcurses"
+  url "https://github.com/dankamongmen/notcurses/archive/refs/tags/v2.3.12.tar.gz"
+  sha256 "ce042908fac11f7df1f9eaa610e46e9c615f53ab036b7c27ae2396292512407b"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "doctest" => :build
+  depends_on "pandoc" => :build
+  depends_on "pkg-config" => :build
+  depends_on "ffmpeg"
+  depends_on "libunistring"
+  depends_on "ncurses"
+  depends_on "readline"
+  uses_from_macos "zlib"
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args + %w[
```

Comment:
```suggestion
    # We pass `-DBUILD_TESTING=on` to build `notcurses-tester`
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args + %w[
```
On the other hand, running `notcurses-tester` just takes a minute or two, so I guess it should be fine. Let's add a comment here as someone (maybe me) is liable to remove `-DBUILD_TESTING=on` somewhere down the road if we don't.

Reply:
i'm happy to switch it up to just be `notcurses-info`, which is much less likely to randomly go bad in the middle of a build. so if the goal is to assure the bare minimum works, `notcurses-info` is the way to go. if we're looking to flush out problems, `notcurses-tester` would be it. it sounds like you're going for the former, and it lets us get rid of `-DBUILD_TESTING=off`.

The only real downside is that `notcurses-tester` will no longer be available for people who want to run it. Oh well, they can build from source.

Reply:
Under what circumstances would a user want to run `notcurses-tester`? If it's genuinely useful to a significant fraction of users, we can build it. If not, then leaving it out is fine.

Reply:
only in the case where they wanted to investigate why the library wasn't working, i.e. if they filed a bug and i asked them to go run `notcurses-tester`. or someone bringing up a new port, i suppose. i think your first inclination was correct, and we oughtn't build it. thanks for the good feedback.

Reply:
Great, sounds good. If someone filed an issue here about something not working, one of the first questions I ask is "can you reproduce this if you install X without using Homebrew?" In which case they'll need to build from source anyway, and they can build `notcurses-tester` then too if they need to.

Reply:
completely agreed